v1/pkg/client/api/openapi.yaml

```diff
@@ -3522,11 +3522,13 @@ components:
           Locations:
           - DataCenter: FTC1
             PrivateNetworkVNISupport: true
+            ControllerID: 046b6c7f-0b8a-43b9-b35d-6489e6daee91
             Region: Central
             ID: 046b6c7f-0b8a-43b9-b35d-6489e6daee91
             PrivateNetworkVIDSupport: true
           - DataCenter: FTC1
             PrivateNetworkVNISupport: true
+            ControllerID: 046b6c7f-0b8a-43b9-b35d-6489e6daee91
             Region: Central
             ID: 046b6c7f-0b8a-43b9-b35d-6489e6daee91
             PrivateNetworkVIDSupport: true
```

Comment:
I know you are just following what is already there, but shouldn't each location in the locations list have a different DataCenter name, ID, and now ControllerID? I don't think it is a real world example to have 2 locations exactly the same. This same comment applies in many places in this file.

Reply:
Yes it is not a real example but the file is generated.